### PR TITLE
[MIG] mgmtsystem_survey: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- mgmtsystem: https://github.com/OCA/management-system/pull/365

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-mgmtsystem @ git+https://github.com/dzungtran89/management-system@14.0-mig-mgmtsystem#subdirectory=setup/mgmtsystem
```
Comment (sebalix) [inline test-requirements.txt]: To remove, merged

**Review by sebalix — COMMENTED** [review]
`pre-commit` is not happy (but CI logs have disappeared) + dependencies to clean up, otherwise LG

File: test-requirements.txt (hunk 1 above)
Comment (dzungtran89) [reply test-requirements.txt]: thx, I updated it

**Review by sebalix — APPROVED** [review]

File: mgmtsystem_survey/migrations/13.0.1.0.0/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        env.cr,
+        """
+        UPDATE survey_survey s
+        SET state = 'management_system'
+        FROM ir_model_data imd
+        WHERE imd.res_id = s.stage_id
+            AND imd.module = 'mgmtsystem_survey'
+            AND imd.name = 'mgmtsystem_survey_type'
+        """,
+    )
```
Comment (ivantodorovich) [inline mgmtsystem_survey/migrations/13.0.1.0.0/pre-migration.py]: Please remove the migrations folder

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#tasks-to-do-in-the-migration

> - Remove any possible migration script from previous version.

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Hi !

Thanks for the migration :+1: 
All looks good except for the `13.0` migration file, could you please remove it?

**Review by simahawk — APPROVED** [review]
LG beside the mig script

**Pull request merged**